addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -132,7 +132,7 @@
                     <BolloVirtuale>SI</BolloVirtuale>
                     <ImportoBollo t-esc="format_numbers(record.l10n_it_stamp_duty)"/>
                 </DatiBollo>
-                <ImportoTotaleDocumento t-esc="format_monetary(record.amount_total, currency)"/>
+                <ImportoTotaleDocumento t-esc="document_total"/>
```

Comment:
I personally prefer `format_monetary(document_total, currency)` here in the template rather than on line 192

Reply:
See the comment from Florian, @FlorianGilbert  --> https://github.com/odoo/odoo/pull/94406/files#r909200978

Reply:
The computations belong in the model.
The formatting belongs in the template.